<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEmployee</class>
 <widget class="QDialog" name="DialogEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>925</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>maanageBill  Employee</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/image/receipt16.png</normaloff>:/icon/image/receipt16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="4" column="0">
    <widget class="QTableWidget" name="tableWidgetEmployee">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Family Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>First Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vat Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Street</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>City</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Country</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Post Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Doy Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mobile</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSalary">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>Salary  :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelTitle">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);
font: 75 15pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>Employee</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonBack">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Back</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/undo24.png</normaloff>:/icon/image/undo24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="labelSelect">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);
font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>Select :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxSearch"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelEnter">
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);
font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>Enter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditEmployee"/>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>S&amp;earch</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/check24.png</normaloff>:/icon/image/check24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="labelEmployee">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;
color:rgb(32, 74, 135)</string>
       </property>
       <property name="text">
        <string>Employee :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonAddEmployee">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>&amp;Add Employee</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonAddSalary">
       <property name="styleSheet">
        <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
       </property>
       <property name="text">
        <string>Add &amp;Salary</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/icon/image/add24.png</normaloff>:/icon/image/add24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;URW Palladio L&quot;;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <column>
      <property name="text">
       <string>Billing Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payment Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bill Paid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
